### PR TITLE
[CoreWLAN] Updates for Xcode14 beta 4

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/CoreWlan/Enums.cs [hunk 1]
```diff
@@ -92,6 +92,10 @@ public enum CWSecurity : ulong {
 		[Mac (10,15)]
 		Wpa3Transition = 13,
 		Unknown = long.MaxValue,
```
Comment (rolfbjarne) [inline src/CoreWlan/Enums.cs:94]: This `Unknown` value should stay at the end (since it's using the last `long` value possible)

**Review by rolfbjarne — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

File: src/CoreWlan/Enums.cs (hunk 1 above)
Comment (tj-devel709) [reply src/CoreWlan/Enums.cs:94]: Good point!

**Review by dalexsoto — APPROVED** [review]

**Review by chamons — APPROVED** [review]

**Comment by tj-devel709** [comment]
Re-kicking for (probably unrelated) Linker web connection failures

**Comment by tj-devel709** [comment]
Unrelated Test Failures: https://github.com/xamarin/maccore/issues/2558

**Pull request merged**